uiowa/uiowa01
# Feature: Collegiate theme remove onecol flex layout

This work relates to issue #240.

# How to test

1. Run drush @theming.local si collegiate
2. Add a basic page
3. Edit the layout
4. Add a section
5. Verify that onecol-flex layout is no longer available
6. Verify that layout descriptions are simplified without "Collegiate" preface.
